lunes_cms/cms/models/document.py

```diff
@@ -51,6 +51,9 @@ class Document(models.Model):
     creation_date = models.DateTimeField(
         auto_now_add=True, verbose_name=_("creation date")
     )
+    definition = models.TextField(max_length=256, blank=True, null=True)
+    additional_meaning_1 = models.CharField(max_length=256, blank=True, null=True)
+    additional_meaning_2 = models.CharField(max_length=256, blank=True, null=True)
```

Comment:
Add `verbose_name` so "Definition" and "Additional meaning 1/2" will be translated into German when German is selected by the user.

```suggestion
    definition = models.TextField(
        max_length=256,
        blank=True,
        null=True,
        verbose_name=_("definition"),
    )
    additional_meaning_1 = models.CharField(
        max_length=256,
        blank=True,
        null=True,
        verbose_name=_("additional meaning 1"),
    )
    additional_meaning_2 = models.CharField(
        max_length=256,
        blank=True,
        null=True,
        verbose_name=_("additional meaning 2"),
    )
```

`verbose_name` should be now included into the migration file. I recommend to delete the atcual one and simply run `./tools/run.sh` to generate a new migration file. (Otherwise `verbose_name` will be added in the second new migration file separately.)